src/state/reducers/notificationReducer.js

```diff
@@ -0,0 +1,23 @@
+import { notifications } from '../actions/index';
```

Comment:
here you import the action type string through the notifications object in ../actions/index - but in 'src/state/actions/notificationActions.js' you specify the notification string explicitly. I recommend that you import notifications the same way in the notificationActions.js file.

Reply:
Good call! I'll update this within my next PR - thanks for catching that.